pcli/src/command/stake.rs

```diff
@@ -242,17 +244,65 @@ impl StakeCmd {
                     total += unbonded.amount;
                 }
 
-                let unbonded = Value {
-                    amount: notes
-                        .get(&*STAKING_TOKEN_DENOM)
-                        .unwrap_or(&BTreeMap::default())
-                        .values()
-                        .flat_map(|notes| notes.iter().map(|n| n.as_ref().amount()))
-                        .sum::<u64>(),
+                let mut unbonded = Value {
+                    amount: 0,
                     asset_id: *STAKING_TOKEN_ASSET_ID,
                 };
 
-                total += unbonded.amount;
+                let mut total_unbonding = Value {
+                    amount: 0,
+                    asset_id: *STAKING_TOKEN_ASSET_ID,
+                };
+                let mut unbonding_timeline = BTreeMap::new();
+
+                // Lifted out because borrow checker
+                let empty_notes = BTreeMap::default();
+
+                for note in notes
+                    .get(&*STAKING_TOKEN_DENOM)
+                    .unwrap_or(&empty_notes)
+                    .values()
+                    .flatten()
+                {
```

Comment:
I don't think this logic is correct, because it assumes that every quarantined note can only hold the staking token, but this is not true (it could also be a different delegation token).